studio-docs/source/org/members.mdx

```diff
@@ -5,53 +5,65 @@ sidebar_title: Members and roles
+
+## Graph-specific member roles
+
+You can override a member's organization-wide role for individual graphs. For example, if a member has the `Observer` role in your organization, you can assign them the `Contributor` role for a graph that they need extra access to.
+
+> A member's graph-specific role must have _more_ permissions than their organization-wide role.
+
+Members with the `Graph Admin` or `Organization Admin` role can assign graph-specific roles from the Access tab of the graph's Settings page.
+
+## Role permissions
+
+> We recently added the `Graph Admin` role and reduced permissions for members with the `Contributor` role.
+
+| Action                                                 | Org Admin | Graph Admin | Billing Manager | Contributor | Observer | Consumer |
+| :----------------------------------------------------- | :-------: | :---------: | :-------------: | :---------: | :------: | :------: |
+| Invite members                                         |     ✓     |             |                 |             |          |          |
+| Remove members                                         |     ✓     |             |        ✓        |             |          |          |
+| View and edit billing information                      |     ✓     |             |        ✓        |             |          |          |
+| Manage organization configuration (name, avatar, etc.) |     ✓     |             |        ✓        |             |          |          |
+| Delete graphs                                          |     ✓     |      ✓      |                 |             |          |          |
+| Create, rename graphs                                  |     ✓     |      ✓      |                 |             |          |          |
+| Manage graph integrations (Datadog, Slack, etc.)       |     ✓     |      ✓      |                 |             |          |          |
+| View and manage graph API keys                         |     ✓     |      ✓      |                 |             |          |          |
+| Modify schema checks configuration                     |     ✓     |      ✓      |                 |             |          |          |
+| Push schemas to your graph                             |     ✓     |      ✓      |                 |      ✓      |          |          |
+| Push schemas to a [protected variant](./graphs/#protected-variants-enterprise-only)                             |     ✓     |      ✓      |                 |            |          |          |
+| Run schema checks                                      |     ✓     |      ✓      |                 |      ✓      |    ✓     |          |
+| View graph usage metrics                               |     ✓     |      ✓      |                 |      ✓      |    ✓     |          |
+| View graph schemas and changelogs                      |     ✓     |      ✓      |                 |      ✓      |    ✓     |    ✓     |
```

Comment:
I think it's probably worth calling out that you need to be an observer to see individual implementing service schemas for federated graphs